src/components/WebexInMeeting/WebexInMeeting.js

```diff
@@ -13,8 +16,15 @@ import './WebexInMeeting.scss';
  * @returns {Object} JSX of the component
  */
 export default function WebexInMeeting({meetingID}) {
+  const [meetingRef, {width}] = useElementDimensions();
+  const cssClasses = classNames({
```

Comment:
![img](https://media3.giphy.com/media/8WOCI38ldkriE/giphy.gif)

Reply:
I don't mind this "object syntax", this is readable lol